lib/vssolution.cpp

```diff
@@ -2204,10 +2204,10 @@ void VisualizationSceneSolution::PrepareLines3()
       if (mesh_coarse)
       {
          auto &ref = mesh->GetRefinementTransforms();
-         //we assume that mesh_course is used only for tensor finite elements,
-         //like for representation of quadratures, so in 2D it is square
+         // we assume that mesh_course is used only for tensor finite elements,
+         // like for representation of quadratures, so in 2D it is square
```

Comment:
I know this is not spelled out, but please try to leave a space after `//` or `///` when commenting for consistency with the existing code 😀

Reply:
Okey dokey 🙃